waku/v2/node/rpc_api/waku_api_rpc.nim

```diff
@@ -0,0 +1,29 @@
+import
+  json_rpc/rpcserver,
+  eth/async_utils,
+  waku_api_tools, waku_api_types,
```

Comment:
prefer explicit `./waku_api_tools` - nim path handling is fickle

Reply:
Addressed in 6bea1b3.